### DISCUSSION TITLE
How to view phone backup albums

### DISCUSSION THREAD

**Discussion by ShiFangJuMie** [immich-app/immich#9042] [Q&A]
Hello everyone

I have successfully backed up my iPhone album to immich, but my other iPhone cannot see this album, and accessing immich through the web cannot see this backed up album

I can see photos in Explore, but I cannot see albums, so I cannot manage and organize these photos.

Using a mobile phone to categorize 3000 photos is a terrible nightmare.

**Comment by bo0tzz** [comment]
This isn't supported yet: #1678
